libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -130,6 +130,11 @@ export class ExecuteBridgeRequest {
         afterResponse:
           command.afterResponse !== undefined ? [command.afterResponse] : [],
       },
+      /*
+       * Reject self-signed and invalid certificates in Production environments but allow them in Development
+       * as it's common for developers to use self-signed certificates in local environments.
+       */
+      rejectUnauthorized: environment.name.toLowerCase() === 'production',
```

Comment:
For later: we need to add and consume an constant value for all of these types Environment checks. We have many of them scattered through the codebase.

Reply:
Totally, and it's not ideal that we rely on the name and not on an environment type.